web/themes/joinup/templates/rdf_entity/rdf-entity--asset-distribution--listing-item.html.twig

```diff
@@ -4,21 +4,34 @@
  * Themeing for rdf entities.
  */
 #}
-<div class="listing__card listing__card--detail-container mdl-card">
+{%
+  set classes = [
+    'listing__card',
+    'mdl-card',
+    'listing__card--detail-container',
+  ]
+%}
+{# @todo Implement changes - https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-3883 #}
```

Comment:
Sorry, but todos are not acceptable. I already marked the ticket as being blocked by those tickets.